src/motion_pvt_sequence_generation/pvt.py

```diff
@@ -0,0 +1,957 @@
+"""
+Helper classes and functions related to PVT.
+
+This file contains sever helper classes for creating and
+manipulating PVT points, segments, and sequences.
+"""
+
+from __future__ import annotations
+import csv
+from dataclasses import dataclass
+from enum import Enum, auto
+
+import numpy as np
+from numpy import float64
+from numpy.typing import NDArray
+from scipy.integrate import quad  # type: ignore
+from scipy.interpolate import splev, splprep  # type: ignore
+from scipy.linalg import solve_banded  # type: ignore
+
+
+@dataclass(frozen=True)
+class Point:
+    """A position-velocity-time point."""
+
+    position: list[float]
```

Comment:
Actually you're right, tuple is a much better choice here (both because we don't users to be able to modify a point once it's created, and because we don't need the overhead of a list.) I've gone ahead and updated these and all other vectors (positions, velocities, accelerations, etc.) in the file accordingly. See 3946ac27f476058d943c7d5b50aedda0eb762267.